dj_secure_settings/loader.py

```diff
@@ -80,7 +82,7 @@ def _load_params_from_yaml(config, yaml_params, env, namespace):
     try:
         for k in yaml_params[env][namespace]:
             config[k] = yaml_params[env][namespace][k]
-    except KeyError:
+    except (KeyError, TypeError):
```

Comment:
The addition of `TypeError` handles a case in which the local yaml file doesn't have a top-level key matching `env`:

```
  File "/Users/elliottyates/at/django-ssm-parameter-store/dj_secure_settings/loader.py", line 81, in _load_params_from_yaml
    for k in yaml_params[env][namespace]:
TypeError: 'NoneType' object is not subscriptable
```